src/builtin/functions/color/opacity.rs

```diff
@@ -5,16 +5,84 @@ use crate::{
     value::Value,
 };
 
+fn is_ms_filter(s: &str) -> bool {
+    let mut chars = s.chars();
+
+    if let Some(c) = chars.next() {
+        if !matches!(c, 'a'..='z' | 'A'..='Z') {
+            return false;
+        }
+    } else {
+        return false;
+    }
+
+    for c in &mut chars {
+        match c {
+            ' ' | '\t' | '\n' => break,
+            'a'..='z' | 'A'..='Z' => continue,
+            '=' => return true,
+            _ => return false,
+        }
+    }
+
+    for c in chars {
+        match c {
+            ' ' | '\t' | '\n' => continue,
+            '=' => return true,
+            _ => return false,
+        }
+    }
+
+    false
```

Comment:
```suggestion
    chars
        .skip_while(|c| matches!(c, 'a'..='z' | 'A'..='Z'))
        .find(|c| !matches!(c, ' ' | '\t' | '\n'))
        .map_or(false, |c| c == '=')
```

Reply:
Hmmm I suppose the code in its original state is unclear. It should be matching the regex `^[a-zA-Z]+\s*=`.

With this suggestion, is that behavior maintained?

Either way I will add a comment explaining what actually constitutes an ms filter match.

Reply:
Yes, the behavior is maintained, the tests passed. First it skips all the ascii alphabetic characters `[a-zA-Z]+`, then it find the non-whitespace `\s*` and check if the next character is `=`. I wonder if it would be simpler if we use regex (I think there might be some performance lost I guess)?

Reply:
Also, after this I will send another suggestion to use `bytes`.

Reply:
Does it still ensure that there is at least one ascii alphabetic character at the start of the string?

Reply:
Ah nevermind I should have been looking at more than just the diff

Reply:
https://github.com/connorskees/grass/pull/30